docs/README.md

```diff
@@ -1,7 +1,159 @@
-# kced temporary documentation
+# Kong go-apiops documentation
 
-Since the `kced` command in intended to be temporary, so is its documentation...
+The [go-apiops](https://github.com/Kong/go-apiops) library provides a set of tools (validation and transformation) for working with API specifications and [Kong Gateway](https://docs.konghq.com/gateway/latest/) declarative configurations. Conceptually, these tools are intended to be organized into a pipeline of individual steps configured for a particular users needs. The overall purpose of the library is to enable users to build a CICD workflow which deliver APIs from specification to deployment. This pipline design allows users to customize the delivery of APIs based on their specific needs.
 
-## command `openapi2kong`
+This project is currently in a public beta preview state. As a result, the tools are available in a temporary command line named `kced`. For installation instructions for the `kced` CLI, see the main [README](../README.md). Before these tools are available in a general release, the tools will be integrated into Kong's declarative management CLI, [deck](https://docs.konghq.com/deck/latest/) and the temporary CLI will be deprecated.
```

Comment:
```suggestion
This project is currently in a public beta preview state. As a result, the tools are available in a temporary command line tool named `kced`. For installation instructions for the `kced` CLI, see the main [README](../README.md). Before these tools are available in a general release, the tools will most likely be integrated into Kong's declarative management CLI, [deck](https://docs.konghq.com/deck/latest/) and the temporary CLI will be deprecated.
```

Reply:
@Tieske we don't need "most likely" here.  This is the plan.  If we need to revise later we can do so inline.

Reply:
updated the suggestion